client/src/components/timeline/user.js

```diff
@@ -18,7 +19,9 @@ const User = ( { user, side, wiki } ) => {
 
 	return (
 		<Header href={href} className="rounded" side={side}>
-			<i className="mr-2 material-icons">person</i>
+			<Link href={href}>
+				<i className="mr-2 material-icons">person</i>
+			</Link>
 			<span>{user}</span>
```

Comment:
The link should be both for the icon as well as the label.

Reply:
I was trying to keep parity with the way the revision icon might work. Is it better for them to be similar in that regard or to have a larger link target for the user?

Reply:
uhhh... I __think__ if there is only one icon/link then the whole icon/label should be linked... now what is the "label" on the revision? you could say it's the username or you could say it's both lines (user & timestamp). Since the icon aligns with the user, I think it's just the username. Though it might be better if the icon was made larger to be the height of both lines. :)

Reply:
I'll play around with it and see what's up. Or, maybe wait on @prtksxna's input before I just make it worse ;)

Reply:
haha, alrighty. :) I can go along with shrinking the link target from the whole area to just the icon + label, but it seems a little aggressive to move it all the way down to just the icon when there still is only a single link. :)